### DISCUSSION TITLE
CentOS 7上编译不过

### DISCUSSION THREAD

**Discussion by ICECHN** [Tencent/libpag#2325] [Q&A] [answered]
在centos 7 上，编译linux目录时报错了，帮忙看一下
![WechatIMG3](https://github.com/Tencent/libpag/assets/11832023/f26872e0-7b1f-482b-a4b8-dd8b72020bd8)

**Comment by domchen** [comment, marked as answer]
注意看Readme文档，GCC要 9.0+以上版本。
